JitsiConference.js

```diff
@@ -964,6 +928,16 @@ JitsiConference.prototype.onMemberLeft = function(jid) {
     }
 };
 
+/**
+ * Method called on local MUC role change.
+ * @param {string} role the name of new user's role as defined by XMPP MUC.
+ */
+JitsiConference.prototype.onLocalRoleChanged = function(role) {
```

Comment:
There are plenty of methods like that which makes it a convention :D for this class.
It feels like they should be non-public with "_" in front of the names. I would rather add a FIXME for all of them as this is a bigger cleanup.